### ISSUE TITLE
Broken links on main page to mailing lists

### ISSUE THREAD

**Issue by jkarinkl** [RIOT-OS/riot-os.org#138]
There are 2 broken links on the RIOT home page under [the section of community](https://www.riot-os.org/#community), "Developers" section:

- [notifications mailing list](http://lists.riot-os.org/mailman/listinfo/notifications/)
- [commits mailing list](http://lists.riot-os.org/mailman/listinfo/commits)

Do these mailing lists still exist? If so, can the links be updated? If not, can these sentences about mailing lists be removed?

**Comment by OlegHahm** [comment]
Good catch and thanks for pointing out. The notifications list does still exist but the link has changed to https://lists.stillroot.org/postorius/lists/notifications.lists.riot-os.org/. The other mailing list does not exist any more. I created a PR (#139) to fix this.

**Comment by miri64** [comment]
#139 was merged a while ago.
